wagtail/wagtailcore/models.py

```diff
@@ -596,6 +598,18 @@ def check(cls, **kwargs):
                 )
             )
 
+        if not accepts_kwarg(cls.relative_url, 'request'):
+            warnings.warn(
+                "%s.relative_url should accept a 'request' keyword argument. "
+                "See http://docs.wagtail.io/en/v1.8/reference/pages/model_reference.html#wagtail.wagtailcore.models.Page.relative_url" % cls,
```

Comment:
This link (and the one below) will now need to point to the (as-yet-unpublished) `v1.10` docs.